aws/kinesis/main.cc

```diff
@@ -118,6 +120,7 @@ void usage(const std::string program_name, const std::string& message) {
     option_description("-l", "--log-level", "Controls the level of detail emitted from the producer.  Valid: ['trace', 'debug', 'info', 'warn', 'error', 'fatal']");
     option_description("-t", "--enable-stack-trace", "Will dump a stack trace and abort if certain memory errors are triggered");
     option_description("-a", "--ca-path", "Location of the CA root certificate that the producer will use for TLS connections.");
+    option_description("-f", "--ca-file", "File of the CA root certificate that the producer will use for TLS connections.");
```

Comment:
We dont need this either, I am not aware of besides KPL java who runs the c++ code directly. So we can skip this for now and add it later if its needed